Tests/test_image_thumbnail.py

```diff
@@ -31,9 +31,9 @@ def test_aspect(self):
         im.thumbnail((100, 50))
         self.assertEqual(im.size, (100, 50))
 
-        im = Image.new("L", (128, 128))
+        im = Image.new("L", (64, 64))
         im.thumbnail((100, 100))
-        self.assertEqual(im.size, (100, 100))
+        self.assertEqual(im.size, (64, 64))
```

Comment:
Why change these lines? Both versions of them pass with or without your source changes.